public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
 
 Special thanks to LynnJones who enhance the readibility, RP for the useful feedback to make it more Charge, and therabidbanana for the progression guideline.
 
+# Party Extra | Extras
+
+An Extra for Charge to add both narrative structure and mechanical benefits to the group of characters represented by the players.
+
+\- By David Haslem
+
+
+## Overview
+
+The group of players is referred to in this extra as a `party`. This group has a separate `party sheet` which tracks `details` of how they are perceived by the world around them, as well as maintaining party-level `danger` and `progress` clocks.
+
+The `party sheet` can represent things such as their “HQ” or “Ship”. It can also be used to represent relationships with important groups or specific NPCs. 
+
+> #### Example
+> 
+> The play group wants to tell the story of a small band of resistance fighters against an evil space empire. They decide they want to start their story with a stolen prototype spaceship and not much else. They have some ideas for characters and they know they want to start on an out-of-the-way space station.
+
+###  Key Concepts and Supporting Details
+
+The `party sheet` tracks four `key concepts`, representing how the party is positioned along four axes. Each `key concept` may have as many as four `supporting details`, which have a `disposition` of positive, negative, or neutral. The combination of these `supporting details` contributes to the overall `key concept` being positive, negative, or neutral.
+
+When `key concepts` or `supporting details` are positive they grant improvements to related rolls the party makes. Negative `key concepts` or `supporting details` cause worse `position` on related rolls.
```

Comment:
question: is only the key concept evaluated when making action rolls ? or can supporting details evaluated on their own?

The phrasing looks a bit weird to me, but I'm having a hard time coming up with something better. Maybe we can ask the server for feedback.

Reply:
I also have some difficulty to understand this part.
I think it'd better to explain the disposition first, then the key concept and the supporting details.

Also, what kind of improvements are granted?

Reply:
My thinking is that `key concepts` deal with broad categories and `supporting details` are very specific. The difference between "+1 effect when dealing with this particular NPC" vs "+1 effect dealing with any friendly NPC". 

Benefits being either position or effect. I'll see how I can rewrite this to communicate that a bit better.

Reply:
Maybe this version better communicates my intent? 

```
The `party sheet` tracks `supporting details` about the party. These details have a `disposition` which might be positive, negative, or neutral. The `disposition` impacts either the `position` or `effect` of rolls related with the specific detail on the `party sheet`.

`Supporting details` are collected into four categories, summarized with four broad `key concepts`. If the majority of details under the `key concept` are positive (or negative), the `disposition` of the `key concept` also becomes positive (or negative). The `disposition` of the `key concept` works the same as `supporting details`, but at a broader scope, impacting all rolls related to the category.
```

Reply:
Perfect, the new version is way better for me.